src/android/data/LocationSerializer.java

```diff
@@ -0,0 +1,47 @@
+package com.tenforwardconsulting.cordova.bgloc.data;
+
+import com.tenforwardconsulting.cordova.bgloc.data.Location;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONException;
+
+/**
+ * Helper class to translate Locations into JSON objects.  The JSON objects aim to mimick the https://developer.mozilla.org/en-US/docs/Web/API/Position API.
```

Comment:
So I think it'd be confusing for consumers to see data in this form that micis the https://developer.mozilla.org/en-US/docs/Web/API/Position object where other data does not.  We at Hurdlr perfer this format because it requires less translation on our end since we're using both navigator.geolocation and this library.